osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/SecurityKeyProvider.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2021 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.factories;
+
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.SecurityKeyType;
+
+/**
+ * Functional interface used in connectors to obtain keys.
```

Comment:
It might be nice, though not required, to document this fact with the java lang `@FunctionalInterface` annotation.

Reply:
Good one. Done.